lib/galaxy/authnz/custos_authnz.py

```diff
@@ -392,9 +392,9 @@ def _validate_nonce(self, trans, nonce_hash):
 
     def _load_config_for_cilogon(self):
         # Set cilogon endpoints
-        self.config["authorization_endpoint"] = "https://cilogon.org/authorize"
-        self.config["token_endpoint"] = "https://cilogon.org/oauth2/token"
-        self.config["userinfo_endpoint"] = "https://cilogon.org/oauth2/userinfo"
+        self.config["well_known_oidc_config_uri"] = self._get_well_known_uri_from_url(self.config["provider"])
```

Comment:
Could you extend the sample config file accordingly? Or is this already covered?

Reply:
It's already covered.